bude_hezky/weather/weather_resolver.py

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/python3
+
+SUNNY_LIKE_CODES = [1, 2, 3, 4, 5, 6, 20, 21, 30]
+
+def is_sunny(forecast_code):
+    return forecast_code in SUNNY_LIKE_CODES
```

Comment:
Jeden soubor na čtyři řádky (který by v týhle aplikaci mohly být i dva, ono to vytvoření seznamu zas tolik nestojí) to je hustý :) Ale nic proti gustu

Reply:
To vzniklo tak, že jsem z toho souboru postupně odmazával a pak to nepřesunul :D Máš pravdu.